documentation/tutorials/!tutorial-template.md

```diff
@@ -0,0 +1,137 @@
+
+For more information on tutorial pages, read the [Good Docs Project guide](https://gitlab.com/tgdp/templates/-/blob/main/tutorial/guide-tutorial.md).
+
+---
+
+<!-- overview -->
+
+In this tutorial, you'll learn how to {insert brief description of the main tutorial task}. This tutorial is intended for {audience}.
+
+By the end of this tutorial, you'll be able to:
+
+- Learning objective 1
+- Learning objective 2
+- Learning objective 3
+
+<!-- background -->
+
+{This section is optional. Feel free to use some of the text below to help you get started.}
+
+- {product} is a {product type} that you can use to {common use case}...
+- Using {feature} enables you to {pain point}...
+
+{Optional: Include links/tooltips to relevant concepts when terms are introduced here or in the `steps` section.}
+
+<!-- prerequisites -->
+
+## Before you begin
+
+{Intro tutorials should have no prerequisites, though more advanced tutorials might have some.}
+
+{Optional: You should be familiar with how to do a [more basic task](/how-to/gwa-install.md) or [concept](/concepts/api-directory.md).}
+
+Before you begin, ensure you:
+
+- [Install gwa CLI](/how-to/gwa-install.md)
+- [Create a Namespace](/resources/gwa-commands.md#namespacecreate)
+
+<!-- steps -->
+
+## {Sub-task name}
+
+{Optional: Provide a concise description of the purpose of this sub-task. Only include this if the purpose is not clear from the sub-task title.}
+
+{You can use this format to describe steps to complete a sub-task:}
+
+1. {Write the first step here. Use a verb to start.}
+
+  {Optional: Explanatory text}
+
+  {Optional: Code sample or screenshot that helps your users complete this step.}
+
+  {Optional: The result of completing this step.}
+
+1. {Write the second step here. Use a verb to start.}
+
+  {Substep 1 text}
+
+  {Substep 2 text}
+
+## {Second sub-task name}
+
+{Repeat as above}
+
+{Optional: Use example code blocks as needed}
+
+{**TO DISCUSS**: When to use placeholders in example code - see [how-to template](/how-to/!how-to-template.md)}
+
+```yaml
+services:
+  - name: my-service
+    host: httpbin.org
+    tags: [ns.<NAMESPACE>]
+    routes:
+      - name: my-service-route
+        tags: [ns.<NAMESPACE>]
+        hosts:
+          - <MYSERVICE>.cluster.local
+```
+
+<!-- cleanup -->
+
+## Clean up
+
+{Optional: Provide instructions on how to clean up the resources resulting from the tutorial.}
+
+{1. On the **Consumers** page, delete any Consumers which were granted access to the service.}
+
+!!! danger
+    This following action is irreversible. Ensure the correct Namespace is active first with:
+    ``` linenums="0"
+    gwa namespace current
+    ```
+
+1. Delete the Namespace and associated services which were set up using during tutorial with this command:
+
+  ``` linenums="0"
+  gwa namespace destroy --force
+  ```
+
+
+<!-- summary -->
+
+## Summary
+
+{Use this section to summarize what the user learned in the tutorial.}
```

Comment:
do we need another summary? it feels repetitive - I would move to the what's next?

also - not sure if this applies to technical documentation but I was told to stay away from questions as headers/titles when writing

Reply:
A tutorial is long and covers many steps. The purpose of tutorials is learning. I think a recap is helpful (think about our Getting Started presentation today).

Re: What's next? - open to other suggestions though I often see `What's next?` used.

Reply:
okay - i think what's next can be replaced by 'further reading' or 'next steps' depending on what is appropriate for the page @rustyjux

Reply:
changed to Next steps.